helpers/config.py

```diff
@@ -1696,44 +1678,23 @@ def __questions_postgres(self):
                 )
                 response = Network.curl(endpoint)
                 if response:
-                    self.__dict['postgres_settings_content'] = re.sub(
-                        r'(log|lc_).+(\n|$)', '', response)
+                    configuration = re.sub(r'(log|lc_).+(\n|$)', '', response)
```

Comment:
I know this isn't being added by this PR, but do we need this step if `include_pgbadger=false`?

Reply:
Right!